integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/search/SearchQueryBaseIT.java

```diff
@@ -38,6 +38,7 @@
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.GenericField;
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
+import org.hibernate.search.util.common.SearchException;
 import org.hibernate.search.util.common.SearchTimeoutException;
 import org.hibernate.search.util.impl.integrationtest.common.rule.BackendMock;
 import org.hibernate.search.util.impl.integrationtest.common.rule.StubSearchWorkBehavior;
```

Comment:
Unrelated to this commit, but I need to put this comment somewhere.

Could you please also add tests for all ID types? I'm thinking of copy/pasting `org.hibernate.search.integrationtest.mapper.pojo.mapping.definition.DocumentIdDefaultBridgeBaseIT#projection` into a new method that uses the `id()` projection instead of the `entityReference()` projection.

Same with `org.hibernate.search.integrationtest.mapper.pojo.mapping.definition.DocumentIdDefaultBridgeOverridingIT#projection`.